builtin/providers/aws/resource_aws_rds_cluster.go

```diff
@@ -372,8 +378,9 @@ func resourceAwsRDSClusterDelete(d *schema.ResourceData, meta interface{}) error
 		DBClusterIdentifier: aws.String(d.Id()),
 	}
 
+	skipFinalSnapshot := d.Get("skip_final_snapshot").(bool)
 	finalSnapshot := d.Get("final_snapshot_identifier").(string)
-	if finalSnapshot == "" {
+	if skipFinalSnapshot == true {
```

Comment:
making `skip_final_snapshot` optional and default false _presumes_ they do not want to skip it, and then assumes they'll supply a `final_snapshot_identifier`, except that attribute is optional as well